docs/feature_combo.md

```diff
@@ -2,6 +2,8 @@
 
 The Combo feature is a chording type solution for adding custom actions. It lets you hit multiple keys at once and produce a different effect. For instance, hitting `A` and `S` within the combo term would hit `ESC` instead, or have it perform even more complex tasks.
 
+Please see [Key Overrides](https://docs.qmk.fm/#/feature_key_overrides?id=difference-to-combos) for multiple keys with modifiers like shift.  
```

Comment:
This isn't exactly clear.

Reply:
Combos don’t work with modifiers like shift or control. Any suggestions?

Reply:
Something along the lines of, "if you're planning on using combos involving modded characters, then key overrides handles this better."